yarn-project/aztec-sandbox/src/sandbox.ts

```diff
@@ -15,22 +24,53 @@ const logger = createDebugLogger('aztec:sandbox');
 
 const localAnvil = foundry;
 
+/**
+ * Helper function that retrieves the addresses of configured deployed contracts.
+ */
+function retrieveL1Contracts(config: AztecNodeConfig, account: Account): Promise<DeployL1Contracts> {
```

Comment:
Took me a while to understand that this function does not _retrieve_ the addresses, it just creates the wallet and public clients and bundles everything together. I'd rename the function, or rather remove it in favor of a helper that just assembles the wallet and public clients (and we reuse that for `deployL1Contracts` as well), and use it along with the `config` to create the `DeployL1Contracts` object directly.